spring-cloud-skipper-server/src/main/java/org/springframework/cloud/skipper/server/config/SkipperServerConfiguration.java

```diff
@@ -46,20 +49,40 @@
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 import org.springframework.core.io.ResourceLoader;
+import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
+import org.springframework.data.map.repository.config.EnableMapRepositories;
 
 /**
  * Main configuration class for the server.
  *
  * @author Mark Pollack
  * @author Ilayaperumal Gopinathan
+ * @author Janne Valkealahti
  */
 @Configuration
 @EnableConfigurationProperties({ SkipperServerProperties.class, CloudFoundryPlatformProperties.class,
 		LocalPlatformProperties.class, KubernetesPlatformProperties.class,
 		MavenConfigurationProperties.class })
 @EntityScan({"org.springframework.cloud.skipper.domain", "org.springframework.cloud.skipper.server.domain"})
+@EnableMapRepositories(basePackages = "org.springframework.cloud.skipper.server.repository")
+@EnableJpaRepositories(basePackages = "org.springframework.cloud.skipper.server.repository")
```

Comment:
We didn't have `EnableJpaRepositories` before. Is this now added for fixing any specific issue?

Reply:
Didn't try any other way as you need to tell where repos as scanned. Previously full boot auto-config did the scanned from app itself.

Reply:
I suspect we may not need this but we can verify this.

Reply:
will verify this while merging.

Reply:
I could confirm the `EnableJpaRepositories` is indeed needed :-)